src/main/java/org/candlepin/subscriptions/db/model/config/AccountConfig.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2009 - 2019 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.db.model.config;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+
+
+/**
+ * Represents the configuration for an account/org pair.
+ */
+@IdClass(AccountConfigKey.class)
+@Entity
+@Table(name = "account_config")
+public class AccountConfig implements Serializable {
```

Comment:
It would be easier to migrate to this if we split into org/account (otherwise we'll need to get the mapping to migrate).

Reply:
I'm assuming you mean something like:

AccountConfig
- account_number
- sync_enabled
- reporting_enabled
- last_updated

OrgConfig
- org_id
- sync_enabled
- last_updated
(no reporting flag required for orgs at this point since conduit doesn't have that concept)

With a repository for each.

Reply:
Perfect! It's like you read my mind :smile: